libraries/RC_Channel/RC_Channels_VarInfo.h

```diff
@@ -90,6 +90,7 @@ const AP_Param::GroupInfo RC_Channels::var_info[] = {
     // @Description: RC input options
     // @User: Advanced
     // @Bitmask: 0:Ignore RC Receiver, 1:Ignore MAVLink Overrides, 2:Ignore Receiver Failsafe bit but allow other RC failsafes if setup, 3:FPort Pad, 4:Log RC input bytes, 5:Arming check throttle for 0 input, 6:Skip the arming check for neutral Roll/Pitch/Yay sticks, 7:Allow Switch reverse, 8:Use passthrough for CRSF telemetry
+    // @Values: 0: None,1:Ignore RC Receiver, 2:Ignore MAVLink Overrides, 4:Ignore Receiver Failsafe bit but allow other RC failsafes if setup, 8:FPort Pad, 16:Log RC input bytes, 32:Arming check throttle for 0 input, 64:Skip the arming check for neutral Roll/Pitch/Yay sticks, 128:Allow Switch reverse, 256:Use passthrough for CRSF telemetry
```

Comment:
An existing problem but it should be "Yaw" instead of "Yay".

As a side note, I have to ask, why on earth is this option here?  Spreading out arming check parameters like this is not a good idea I think.  Users generally expect to use the ARMING_CHECKs to decide what is checked or not checked.

Reply:
fixed...fyi, here is the discussion re the arming checks that you participated in...and had reservations then apparently...